TypeScript.py

```diff
@@ -792,6 +795,29 @@ def on_close(self, view):
                     cli.service.close(view.file_name())
         logger.view_debug(view, "exit on_close")
 
+    def handlePulse(self):
+        view = active_view()
+        info = self.getInfo(view)
+        if info:
+            self.showTopLineStatus(info, view)
+        sublime.set_timeout(self.handlePulse, 500)
```

Comment:
Problem is there is no way to detect scrolling.  Not even a text command or selection modified event.  Would love to just move the icon upon scrolling, with no timer loop.  I am open to other approaches that would not involve a gutter icon.   Don't particularly like the timer loop.